talk/basicconcepts/control.tex

```diff
@@ -75,13 +75,13 @@
   \frametitlecpp[98]{Control structures: switch}
   \begin{block}{Syntax}
     \begin{cppcode*}{gobble=2}
-      switch(identifier) {
-        case c1 : statements1; break;
-        case c2 : statements2; break;
-        case c3 : statements3; break;
-        ...
-        default : instructiond; break;
-      }
+        switch(identifier) {
+          case c1 : statements1; break;
+          case c2 : statements2; break;
+          case c3 : statements3; break;
+          ...
+          default : instructiond; break;
+        }
```

Comment:
I think you could do the same using `gobble=0`. To be tried.

Reply:
I will rebase and use `gobble=0` instead.